includes/apple-exporter/components/class-body.php

```diff
@@ -111,9 +113,39 @@ public function register_specs() {
 		$this->register_spec(
 			'default-body',
 			__( 'Default Style', 'apple-news' ),
-			$this->get_default_style_spec()
+			$default_spec
 		);
 
+		$dropcap_color_dark            = $theme->get_value( 'dropcap_color_dark' );
+		$dropcap_background_color_dark = $theme->get_value( 'dropcap_background_color_dark' );
+
+		$dark_colors_exist = ! empty( $dropcap_color_dark ) || ! empty( $dropcap_background_color_dark );
+
+		$conditional = array();
+		if ( $dark_colors_exist ) {
+			$conditional = array(
+				'conditional' => array_merge(
+					array(
+						'dropCapStyle' => array(
+							'numberOfLines' => '#dropcap_number_of_lines#',
+						),
+						'conditions'   => array(
+							'preferredColorScheme' => 'dark',
+						),
+					),
+					$default_spec['conditional']
+				),
+			);
+		}
+
+		if ( ! empty( $dropcap_color_dark ) ) {
```

Comment:
If the dark color doesn't exist for one of these options, but it does for the other, does it fall back to the non-dark mode color?

Reply:
Yup :)